core/src/main/java/com/graphhopper/routing/ev/Country.java

```diff
@@ -242,7 +245,7 @@ public enum Country {
     ZMB("Zambia", "ZM"),
     ZWE("Zimbabwe", "ZW");
 
-    public static final String KEY = "country";
+    public static final String KEY = "country", JSON_ID = "id";
```

Comment:
We now use two different names for the same thing? We have a `twoLetterCode` field in `Country` (and the corresponding `getTwoLetterCode`. And then there is `Country#valueOfAlpha2` which expects as input such a `twoLetterCode`.